errors/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/rollbar/rollbar-go/errors
 
+go 1.13
+
```

Comment:
This was added automatically since I'm using Go 1.13, but it shouldn't prevent older versions of Go from using this module. See https://golang.org/doc/go1.12#modules.